index.html

```diff
@@ -1989,7 +1989,7 @@
 <p>
   <span class="practicelab" id="principle-minimize-consent-requests">
     An [=actor=] should avoid interrupting a [=person=]'s use of a site for
-	  consent requests when an alternative is available.
+          consent requests when an alternative is available.
```

Comment:
While I agree with this particular whitespace change (down with tabs!), it's a shame when such changes are mixed up with substantive changes.